frontend/src/routes/index.svelte

```diff
@@ -59,6 +68,7 @@
 			<ScriptGettingStarted />
 
 			<div class="mt-6 mb-2 text-md font-bold text-gray-900 ">Latest scripts:</div>
+			<Skeleton loading={loading.scripts} layout={[1, [12.25, 12.25, 12.25]]} />
```

Comment:
nit: those skeleton cards are slightly too tall on my end.

Reply:
How tall are the cards on your screen?